pkg/webhook/admission/builder.go

```diff
@@ -61,21 +65,30 @@ func NewWebhookBuilder() *WebhookBuilder {
 	return &WebhookBuilder{}
 }
 
-// Name sets the name of the server.
+// Name sets the Name of the webhook.
 // This is optional
 func (b *WebhookBuilder) Name(name string) *WebhookBuilder {
 	b.name = name
 	return b
 }
 
+// Annotations sets the annotations of the webhook.
+// This annotation will be passed to the handler along with each incoming request.
+// This annotations is not the .metadata.annotations in kubernetes.
+// This is optional
+func (b *WebhookBuilder) Annotations(anno map[string]string) *WebhookBuilder {
```

Comment:
we should consider naming this something else because Annotations is a well known term in kubernetes ?